File: solucoes/2009/nivel jr/fase 1/py/overflow.py
V = int(input())
P, C, Q = list(map(str, input().split()))

if C == '*':
    resultado = int(P) * int(Q)

    if resultado > V:
        print('OVERFLOW')
    else:
        print('OK')
else:
    resultado = int(P) + int(Q)

    if resultado > V:
        print('OVERFLOW')
    else:
        print('OK')

File: solucoes/2009/nivel jr/fase 1/js/overflow.js
var V
var P, C, Q

scanf("%d", "V")
scanf("%d%s%d", "P", "C", "Q")

if(C == '*'){
    var resultado = int(P) * int(Q)

    if(resultado > V){
        printf("%s", 'OVERFLOW')
    }
    else{
        printf("%s", 'OK')
    }
}
else{
    var resultado = P + Q

    if(resultado > V){
        printf("%s", 'OVERFLOW')
    }
    else{
        printf("%s", 'OK')
    }
}